### DISCUSSION TITLE
Prevent AI assistant from renaming variables

### DISCUSSION THREAD

**Discussion by tennox** [windmill-labs/windmill#4381] [Questions]
The AI assistant is a bit too opinionated imho, e.g. it always renames my variables to add `Lib` and `Resource` suffixes - can I prevent that? (changing prompt?)

![Screenshot from 2024-09-12 13-18-05](https://github.com/user-attachments/assets/f7b97fee-bf57-4770-899b-dbd31f99ed87)

**Comment by rubenfiszel** [comment]
At the very least you can play and try to improve our prompts here: https://github.com/windmill-labs/windmill/blob/main/frontend/src/lib/components/copilot/prompts/fix.yaml
